eppo_core/src/precomputed.rs

```diff
@@ -99,23 +130,42 @@ impl PrecomputedConfiguration {
 
 impl From<PrecomputedConfiguration> for ObfuscatedPrecomputedConfiguration {
     fn from(config: PrecomputedConfiguration) -> Self {
-        let salt = rand::thread_rng().gen();
+        let salt: Str = {
+            let bytes = rand::thread_rng().gen::<[u8; 16]>();
+            base64::prelude::BASE64_STANDARD_NO_PAD
+                .encode(&bytes)
+                .into()
+        };
         ObfuscatedPrecomputedConfiguration {
             obfuscated: serde_bool::True,
             format: ConfigurationFormat::Precomputed,
-            salt,
             created_at: config.created_at,
             environment: config.environment,
             flags: config
                 .flags
                 .into_iter()
                 .map(|(k, v)| {
                     (
-                        Md5HashedStr::new(&salt, k.as_bytes()),
+                        Md5HashedStr::new(salt.as_bytes(), k.as_bytes()),
                         ObfuscatedPrecomputedAssignment::from(v),
                     )
                 })
                 .collect(),
+            bandits: config
```

Comment:
do you have the bandit obfuscation as part of a unit test?

Reply:
There is a small test to compute bandits against `bandit-flags-v1.json`+`bandit-models-v1.json` and I verified the output.

Overall, I'm more reliant on types here rather than tests. Having obfuscated type clearly define where md5s and base64s go makes it trivial to ensure correctness. (If the type requires an md5, there's only one way to get — hash salt+string — regular strings and base64 don't fit.)

Once we stabilize formats, I do want to add a couple of test cases to sdk-test-data though